wagtail/admin/templates/wagtailadmin/account/settings_panels/avatar.html

```diff
@@ -1,35 +1,16 @@
 {% load i18n wagtailadmin_tags %}
 {% block content %}
-    <div class="avatar-panel">
-        <div class="avatar-panel__image">
-            {% avatar user=request.user size="large" %}
+    <div class="avatar-panel w-flex w-flex-wrap w-items-center w-mt-5">
+        <div class="avatar-panel__image w-mr-9 w-mb-5">
+            {% if form.avatar.value %}
+                <input class="avatar-clear" type="checkbox" name="avatar-avatar-clear" id="avatar-avatar-clear_id">{% trans "Reset to default" %}
+                <img class="avatar large gravatar w-block w-mt-3" src="{% avatar_url request.user 70 gravatar_only=True %}" alt="default avatar image"/>
+            {% endif %}
+            {% avatar user=request.user classname="avatar-span w-block w-mt-3" size="large" %}
         </div>
         <div class="avatar-panel__form">
             <label for="id_avatar-avatar">{% trans "Upload a profile picture:" %}</label>
             <input type="file" name="avatar-avatar" accept="image/*" id="id_avatar-avatar">
-            <input type="hidden" name="avatar-avatar-clear" id="avatar-avatar-clear_id">
-
-            {% if form.avatar.value %}
-                <button type="button" class="avatar-panel__revert-to-gravatar button button-secondary" data-gravatar-url="{% avatar_url request.user 70 gravatar_only=True %}">{% trans "Reset to default" %}</button>
-            {% endif %}
         </div>
     </div>
-
-    <script>
-        (function () {
-            var revertToGravatarButton = document.querySelector('.avatar-panel__revert-to-gravatar');
-            if (revertToGravatarButton) {
-                revertToGravatarButton.addEventListener('click', function(e) {
-                    e.preventDefault();
-
-                    var avatarImagePreview = document.querySelector('.avatar-panel__image img');
-                    avatarImagePreview.src = revertToGravatarButton.dataset.gravatarUrl;
-
-                    document.getElementById('avatar-avatar-clear_id').value = 'on';
-
-                    revertToGravatarButton.remove();
-                });
-            }
-        })();
-    </script>
-{% endblock %}
+{% endblock %}
```

Comment:
Be careful with removing line breaks at the end of the file, you may need to install a VSCode plugin like this https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

See the diff it has the `(-)` which indicates that a line break at the end of the file was removed.
```suggestion
{% endblock %}

```